.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea
+glenrg.crt
```

Comment:
Please don't include personal files here. Instead, set up a global .gitignore file on your machine to catch these things. 

```
echo "glenrg.crt" > ~/.gitignore_global
git config --global core.excludesfile ~/.gitignore_global
```

Then, you don't have to have commits for your own personal files in the history.

Reply:
NB: I was only refering to glenrg.crt; .idea is pretty shared, so can stay :)